src/rust/engine/fs/src/store.rs

```diff
@@ -2001,23 +2029,28 @@ mod remote {
       request: bazel_protos::remote_execution::FindMissingBlobsRequest,
     ) -> impl Future<Item = HashSet<Digest>, Error = String> {
       let store = self.clone();
-      self.with_cas_client(move |client| {
-        client
-          .find_missing_blobs_opt(&request, store.call_option())
-          .map_err(|err| {
-            format!(
-              "Error from server in response to find_missing_blobs_request: {:?}",
-              err
-            )
-          })
-          .and_then(|response| {
-            response
-              .get_missing_blob_digests()
-              .iter()
-              .map(|digest| digest.into())
-              .collect()
-          })
-      })
+      let metadata =
+        ContentRequestMetadata::new(format!("finding missing blobs for {:?}", request.clone()));
```

Comment:
You shouldn't need to clone things into format args, as they're always taken as references